data-analysis/template.yaml

```diff
@@ -511,6 +511,23 @@ Resources:
       QueueName: !Sub ${AWS::StackName}-${Environment}-query-completed-dead-letter-queue
       KmsMasterKeyId: !GetAtt QueryCompletedQueueKmsKey.Arn
 
+  ################
+  # Email to Slack
+  ################
+
+  EmailToSlackSNSTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: !Sub '${AWS::StackName}-email-to-slack-topic'
```

Comment:
Slightly picky, but calling it "email to slack" is maybe not a great name, because what you actually want here is something that lets somebody send an SNS topic message, and it goes into Slack. The fact that you're doing this via email is kind of irrelevant, I think?

Reply:
yeah, im not the best at naming things :/ but i like it